pkg/controllers/networkinfo/networkinfo_utils.go

```diff
@@ -83,12 +84,12 @@ func setVPCNetworkConfigurationStatusWithLBS(ctx *context.Context, client client
 			return
 		}
 	}
-	// else append the new VPCInfo
-	if nc.Status.VPCs == nil {
-		nc.Status.VPCs = []v1alpha1.VPCInfo{}
-	}
```

Comment:
To double confirm with you, line 86-89 are not necessary for the first time CR status update?

Reply:
AFAIK, even a `nil` slice can be operated on with `append`, so there's no need for an additional nil check.